packages/globals/src/globals.ts

```diff
@@ -80,7 +99,12 @@ export function setInteractorTimeout(ms: number): void {
 }
 
 export function addActionWrapper<T>(
-  wrapper: (description: string, action: () => Promise<T>, type: ActionType) => () => Promise<T>
+  wrapper: (
+    description: string,
+    action: () => Promise<T>,
+    type: ActionType,
+    options: InteractorOptions
+  ) => () => Promise<T>
 ): () => boolean {
```

Comment:
It looks like this wrapper should be a type:

```ts
export interface InteractionEvent<T> {
  description: string;
  action: () => Promise<T> ;
  type: ActionType;
  options: InteractionOptions;
}

export type InteractionHook<T> = (event: InteractionEvent<T>) => Promise<T>;

export function addActionWrapper<T>(hook: InteractionHook<T>): () => boolean {
  // add it
}
```

Reply:
Done. That was a little bit tricky to not break things